miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditKitDescriptorController.java

```diff
@@ -113,12 +119,14 @@ public ModelAndView setupForm(@PathVariable Long kitDescriptorId, ModelMap model
   }
 
   @RequestMapping(method = RequestMethod.POST)
-  public String processSubmit(@ModelAttribute("kitDescriptor") KitDescriptor kitDescriptor, ModelMap model, SessionStatus session)
+  public String processSubmit(@ModelAttribute("kitDescriptor") KitDescriptor kitDescriptor, ModelMap model, SessionStatus session,
+      UriComponentsBuilder uriBuilder, HttpServletResponse response)
```

Comment:
Added `uriBuilder` and `response` parameters no longer necessary. Same in other controllers